reprohack_hub/static/vendor/node_modules/@fortawesome/fontawesome-free/package.json

```diff
@@ -1,58 +1,87 @@
 {
-  "description": "The iconic font, CSS, and SVG framework",
-  "keywords": [
-    "font",
-    "awesome",
-    "fontawesome",
-    "icon",
-    "svg",
-    "bootstrap"
+  "_args": [
+    [
+      "@fortawesome/fontawesome-free@5.15.4",
+      "/Users/Anna/Documents/workflows/Reprohack-org/reprohack_site"
+    ]
   ],
-  "homepage": "https://fontawesome.com",
-  "bugs": {
-    "url": "http://github.com/FortAwesome/Font-Awesome/issues"
+  "_from": "@fortawesome/fontawesome-free@5.15.4",
+  "_id": "@fortawesome/fontawesome-free@5.15.4",
+  "_inBundle": false,
+  "_integrity": "sha512-eYm8vijH/hpzr/6/1CJ/V/Eb1xQFW2nnUKArb3z+yUWv7HTwj6M7SP957oMjfZjAHU6qpoNc2wQvIxBLWYa/Jg==",
+  "_location": "/@fortawesome/fontawesome-free",
+  "_phantomChildren": {},
+  "_requested": {
+    "type": "version",
+    "registry": true,
+    "raw": "@fortawesome/fontawesome-free@5.15.4",
+    "name": "@fortawesome/fontawesome-free",
+    "escapedName": "@fortawesome%2ffontawesome-free",
+    "scope": "@fortawesome",
+    "rawSpec": "5.15.4",
+    "saveSpec": null,
+    "fetchSpec": "5.15.4"
   },
+  "_requiredBy": [
+    "/"
+  ],
+  "_resolved": "https://registry.npmjs.org/@fortawesome/fontawesome-free/-/fontawesome-free-5.15.4.tgz",
+  "_spec": "5.15.4",
+  "_where": "/Users/Anna/Documents/workflows/Reprohack-org/reprohack_site",
   "author": {
     "name": "Dave Gandy",
     "email": "dave@fontawesome.com",
-    "web": "http://twitter.com/davegandy"
+    "url": "http://twitter.com/davegandy"
+  },
+  "bugs": {
+    "url": "http://github.com/FortAwesome/Font-Awesome/issues"
   },
   "contributors": [
     {
       "name": "Brian Talbot",
-      "web": "http://twitter.com/talbs"
+      "url": "http://twitter.com/talbs"
     },
     {
       "name": "Travis Chase",
-      "web": "http://twitter.com/supercodepoet"
+      "url": "http://twitter.com/supercodepoet"
     },
     {
       "name": "Rob Madole",
-      "web": "http://twitter.com/robmadole"
+      "url": "http://twitter.com/robmadole"
     },
     {
       "name": "Geremia Taglialatela",
-      "web": "http://twitter.com/gtagliala"
+      "url": "http://twitter.com/gtagliala"
     },
     {
       "name": "Mike Wilkerson",
-      "web": "http://twitter.com/mw77"
+      "url": "http://twitter.com/mw77"
     }
   ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/FortAwesome/Font-Awesome"
-  },
+  "dependencies": {},
+  "description": "The iconic font, CSS, and SVG framework",
   "engines": {
     "node": ">=6"
   },
-  "dependencies": {},
-  "version": "5.15.4",
-  "name": "@fortawesome/fontawesome-free",
-  "main": "js/fontawesome.js",
-  "style": "css/fontawesome.css",
```

Comment:
@twinkarma , am I overwriting the local version you had installed here? If so how can I guard against this when upgrading npm packages?

Reply:
Sorry, not sure what you're trying to do there? I wouldn't go changing the package.json files of your downloaded libraries,it will get overwritten anyway if anyone decide to reinstall npm packages .

Reply:
So this was automatic (ie I didn't make it manually) and the result of running:
```
npm audit fix
npm install
npm run serve
```

Should I remove the change to this file from this PR then?

Reply:
The file does not actually get used in production. You can even delete it if you want. It'll just get overwritten the next time someone does a rebuild/reinstall of the node libraries and tries to commit it to repo.